src/lib.rs

```diff
@@ -371,246 +493,346 @@ struct Category<BlockNumber, Moment, AccountId> {
     forum_sudo_creator: AccountId
```

Comment:
It could be just `owner_id` or `creator_id`. I mean that in future different user roles could be able to create categories. But calling this `forum_sudo_creator` restricts it to the forum sudo.

Reply:
This was fixed in response to a review of the PR #3 

https://github.com/Joystream/substrate-forum-module/pull/3/commits/0795f21083b0ba8df768faa09ebd321ed062b938